main.py

```diff
@@ -363,9 +407,17 @@ def draw_game(self):
             pyxel.bltm(0, 0, 1, scroll_x, 0, 128, 128, 0)
             pyxel.camera(scroll_x, 0)
             self.player.draw()
+            for i in enemies:
+                i.draw()
+            if self.paused:
+                display_text(adjust_x(50), 40, "PAUSED!")
+                display_text(adjust_x(30), 50, "Press R to Restart!")
+                display_text(adjust_x(30), 60, "Press P to Resume!")
+                display_text(adjust_x(30), 70, "Press Q to Quit!")
         else:
             display_text(adjust_x(30), 30, "Oh no! You lost!")
-            display_text(adjust_x(0), 40, "Press Q to quit, and try again!")
+            display_text(adjust_x(15), 40, f"Press R to restart, and\n{' '*6}try again!")
```

Comment:
This is what Black suggested me:

```suggestion
            display_text(
                adjust_x(15), 40, f"Press R to restart, and\n{' '*6}try again!"
            )
```

Reply:
CRazy suggestions 😆 ... that's the 80 characters line width I guess...